src/components/ListWraper/ListWraper.js

```diff
@@ -2,31 +2,31 @@ import React, { Component } from 'react';
 import ListItem from './ListItem/ListItem';
 
 class ListWrapper extends Component {
- 
-  render(){
+
+  render() {
     const { header, users } = this.props.items;
```

Comment:
itemsy też sie da zdestrukutryzować. Krzyu zwróć uwagę zę wsze∂żie gdzie masz dwie kropki i trzy słowa to już jest za dużo. to znaczy że da sie to zdesktrukturyzować.

`const {items: {header, users}} = this.props;`